nca/CoreDS/ConnectionSet.py

```diff
@@ -550,3 +570,109 @@ def get_non_tcp_connections():
         res.add_all_connections([ProtocolNameResolver.get_protocol_number('TCP')])
         return res
         # return ConnectionSet(True) - ConnectionSet.get_all_TCP_connections()
+
+    # TODO - after moving to the optimized HC set implementation,
+    #  get rid of ConnectionSet and move the code below to ConnectivityProperties.py
+    @staticmethod
+    def conn_props_to_fw_rules(conn_props, cluster_info, peer_container, ip_blocks_mask,
+                               connectivity_restriction):
+        """
+        Build FWRules from the given ConnectivityProperties
+        :param ConnectivityProperties conn_props: properties describing allowed connections
+        :param ClusterInfo cluster_info: the cluster info
+        :param PeerContainer peer_container: the peer container
+        :param IpBlock ip_blocks_mask: IpBlock containing all allowed ip values,
+         whereas all other values should be filtered out in the output
+        :param Union[str,None] connectivity_restriction: specify if connectivity is restricted to
+               TCP / non-TCP , or not
+        :return: FWRules map
+        """
+        ignore_protocols = ProtocolSet()
```

Comment:
rename `ignore_protocols` to `relevant_protocols`

Reply:
Done